src/kv/RocksDBStore.cc

```diff
@@ -739,6 +742,8 @@ int RocksDBStore::get(
   s = db->Get(rocksdb::ReadOptions(), rocksdb::Slice(k), &value);
   if (s.ok()) {
     out->append(value);
+  } else if (s.IsIOError()) {
```

Comment:
i mean, we can take this opportunity to be more explicit about handling `-ENOENT`.